wiki/Tournaments/The_Roundtable/16_player_battle_royale/en.md

```diff
@@ -0,0 +1,64 @@
+---
+tags:
+  - The Round table
+---
+
+# The Roundtable 16 player battle royale
+
+![The Roundtable banner](/wiki/shared/news/2022-07-08-the-roundtable/roundtable_banner.png)
+
+[**The Roundtable**](/wiki/Tournaments/The_Roundtable)'s **16 player battle royale** takes place the first day of the event.
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Unique profile badge |
+
+## Participants
+
+| User |
+| :-- |
+| ![][flag_AU] [sakamata1](https://osu.ppy.sh/users/7562902) |
+| ![][flag_US] [Kariyu](https://osu.ppy.sh/users/4733121) |
+| ![][flag_CA] [RyuK](https://osu.ppy.sh/users/6304246) |
+| ![][flag_US] [Trail Mix](https://osu.ppy.sh/users/3328742) |
+| ![][flag_US] [im a fancy lad](https://osu.ppy.sh/users/4908650) |
+| ![][flag_US] [fieryrage](https://osu.ppy.sh/users/3533958) |
+| ![][flag_US] [Utami](https://osu.ppy.sh/users/7512553) |
+| ![][flag_US] [Gurbzy](https://osu.ppy.sh/users/12970596) |
+| ![][flag_US] [Apraxia](https://osu.ppy.sh/users/4194445) |
+| ![][flag_US] [headstack](https://osu.ppy.sh/users/10322186) |
+| ![][flag_US] [Mlaw](https://osu.ppy.sh/users/3126596) |
+| ![][flag_US] [kablaze](https://osu.ppy.sh/users/3043603) |
+| ![][flag_US] [Sytho](https://osu.ppy.sh/users/4175698) |
+| ![][flag_US] [KoaLeahq](https://osu.ppy.sh/users/4321876) |
+| ![][flag_US] [Mathyu](https://osu.ppy.sh/users/6303313) |
+| ![][flag_CA] [Rupertion](https://osu.ppy.sh/users/5774823) |
+
+## Ruleset
+
+### General rules
+
+- This tournament is a 16 player osu! battle royale style tournament.
+- Beatmap scoring will be based on Score V2.
+
+### Battle royale rules
+
+The battle royale will proceed as follows:
+
+- 16 players will be randomized into two groups of 8.
+- Each group will play through 7 maps. After each map, the lowest scoring user will be eliminated.
+- After both groups are narrowed down to one player each, the final players will compete in a BO13 1v1 match.
+
+### 1v1 rules
+
+- When both players are in the room and ready, the match will start.
+W- hen the match stats, both users will `!roll` to determine the pick and ban order. The roll winner will choose either the pick order or the ban order, and roll loser will choose the remaining order. Bans will be done in ABBA order.
```

Comment:
- When the match starts,

Reply:
@Blooshing please use the suggestion button to provide suggestion for changes

![Safari 2022-07-08 at 05 28 54](https://user-images.githubusercontent.com/191335/177923295-5c4da8e5-90ce-4a22-993b-3914082b2348.png)

```suggestion
- When the match stats, both users will `!roll` to determine the pick and ban order. The roll winner will choose either the pick order or the ban order, and roll loser will choose the remaining order. Bans will be done in ABBA order.
```